content/docs/thinking-in-react.md

```diff
@@ -8,13 +8,13 @@ redirect_from:
 prev: composition-vs-inheritance.html
 ---
 
-برأينا (React) هو الطريقة الاساسية لبناء تطبيقات ويب كبيرة وسريعة بإستخدام الجافاسكريبت (Javascript) لقد ساعدنا في التوسع في التطبيقات بشكل جيد بفيسبوك (Facebook) وانستجرام (Instagram).
+برأينا React هو الطريقة اﻷساسية لبناء تطبيقات ويب كبيرة وسريعة باستخدام Javascript، لقد ساعدنا في التوسع في التطبيقات بشكل جيد بفيسبوك (Facebook) وانستجرام (Instagram).
```

Comment:
you don't have to double write the word twice in Arabic and English like Facebook and Instagram just translate them into arabic only, they are very common in Arabic right now

Reply:
I'm not the translator, I'm just a reviewer, I think we should not write them in Arabic!
@3imed-jaberi your thoughts?